ffc-node-development/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG REGISTRY=562955126301.dkr.ecr.eu-west-2.amazonaws.com
+ARG BASE_VERSION=1.0.0
+FROM $REGISTRY/ffc-node-base:$BASE_VERSION
+ARG VERSION=1.0.0

```

Comment:
Has there been much discussion about how to version parent images? I can see this getting quite confusing if the development and base image versioning diverges. Developers extending these images need a way to identify which base image version a given development image came from.

Adding a base version label could help with that, although it might be nicer to simply enforce that the versions must always match and build the images as a pair whenever either Dockerfile changes.

Reply:
I've removed the second DEV_VERSION setting, so that the BASE_VERSION is used in both images.